src/app.rs

```diff
@@ -175,6 +181,21 @@ pub fn app() -> App<'static, 'static> {
         .help("Directory containing the Unicode character database files.");
 
     // Subcommands.
+    let cmd_bidi_mirroring_glyph =
+        SubCommand::with_name("bidi-mirroring-glyph")
+            .author(crate_authors!())
+            .version(crate_version!())
+            .template(TEMPLATE_SUB)
+            .about("Create Unicode Bidi Mirroring Glyph table.")
+            .before_help(ABOUT_BIDI_MIRRORING_GLYPH)
+            .arg(ucd_dir.clone())
+            .arg(flag_fst_dir.clone())
+            .arg(flag_name("BIDI_MIRRORING_GLYPH"))
+            .arg(flag_chars.clone())
+            .arg(flag_trie_set.clone())
+            .arg(Arg::with_name("rust-match").long("rust-match").help(
+                "Emit a function that uses a match to map between codepoints.",
```

Comment:
Depending on what we decide below, at least as of know, I might also add, "with unknown codepoints mapping to 0."